### PR TITLE
Removed the code related to PHP templates

### PR DESCRIPTION
**Pull request by javiereguiluz**
As we've mentioned in some of the last pull requests, our resources to maintain Symfony Docs are very limited and we're testing ideas to improve the situation.

That's why we propose to remove all the examples about PHP templates and keep only the Twig examples. The developers who still want/must use PHP templates, can read our article about that: https://symfony.com/doc/current/templating/PHP.html

If this idea is approved, I'll create a separate PR for each maintained branch, to avoid merging branches up.

PS: this idea is limited to PHP and Twig. We are not planning to remove all the `configuration-block` directives. We'll keep documenting the config in YAML, PHP and XML formats.

### CODE REVIEW

**Comment by javiereguiluz** [comment]
I've removed more PHP code blocks not found before (because the RS code directives were a bit different) and I've moved the PHP code related to forms to the main PHP article.

**Review by xabbuh — APPROVED** [review]

File: form/form_customization.rst [hunk 1]
```diff
@@ -348,55 +288,10 @@ different than the one of your main form. Just specify both your themes:
...
-.. note::
-
-    The ``:form`` syntax is based on the functional names for templates:
-    ``Bundle:Directory``. As the form directory lives in the
-    ``app/Resources/views`` directory, the ``Bundle`` part is empty, resulting
-    in ``:form``.
-
-Referencing base Form Blocks (Twig specific)
---------------------------------------------
+Referencing base Form Blocks
```
Comment (xabbuh) [inline form/form_customization.rst]: we should add a label for the old headline

File: templating/PHP.rst [hunk 2]
```diff
@@ -383,4 +383,224 @@ instance, to output something in a JavaScript script, use the ``js`` context::
...
+(``*``), you can do this by customizing the ``form_label`` fragment. When using
+PHP as a templating engine you have to copy the content from the original
+template:
+
+.. code-block:: html+php
+
+    <!-- form_label.html.php -->
+
+    <!-- original content -->
+    <?php if ($required) { $label_attr['class'] = trim((isset($label_attr['class']) ? $label_attr['class'] : '').' required'); } ?>
```
Comment (HeahDude) [inline templating/PHP.rst]: Missing `<?php if (false !== $label) : ?>` at first and the corresponding `<?php endif ?>` at last.

**Comment by javiereguiluz** [comment]
Closing because of the conflicts and because I did too many unrelated changes. I'll redo this PR in the future. Thanks!

**Pull request closed without merging**